.github/workflows/hugegraph-llm.yml

```
@@ -0,0 +1,199 @@
+        else
+          echo "✗ No dependency files found!"
+          exit 1
+        fi
+        
+    # Verify and complete package installation
+    - name: Verify installation
+      run: |
+        source .venv/bin/activate
+        echo "=== Installing hugegraph-python-client ==="
+        uv pip install -e ./hugegraph-python-client/
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)